src/core/algorithms/statistics/statistic.h

```diff
@@ -13,6 +13,7 @@ class Statistic {
 public:
     Statistic() noexcept = default;
     Statistic(std::byte const *, model::Type const *, bool clone_data);
+    Statistic(std::byte const *, model::TypeId, bool clone_data = false);
```

Comment:
Don't omit parameter names, this impairs readability.

Reply:
Tbh I don't remember at all what all this is about, is this constructor going to be used anywhere?

Reply:
For some reason new implemented method doesn't pass python bindings tests, though passes all cpp tests.